semantic-go/src/Language/Go/AST.hs

```diff
@@ -18,4 +18,4 @@ import           Prelude hiding (False, Float, Integer, Rational, String, True)
 import           AST.GenerateSyntax
 import qualified Language.Go.Grammar as Grammar
 
-astDeclarationsForLanguage Grammar.tree_sitter_go "../../../vendor/tree-sitter-go/src/node-types.json"
\ No newline at end of file
+astDeclarationsForLanguage Grammar.tree_sitter_go "../../../vendor/tree-sitter-go/src/node-types.json"
```

Comment:
@aymannadeem didn’t catch this earlier, but you should probably configure your editor to insert newlines at EOF. (I’m actually surprised it doesn’t already.)